src/types/instantsearch.ts

```diff
@@ -13,16 +13,15 @@ export type HelperChangeEvent = {
 
 type HitAttributeHighlightResult = {
   value: string;
-  matchLevel: 'none' | 'partial' | 'full';
+  matchLevel: string;
```

Comment:
Because `HitAttributeHighlightResult` was not exported, this enumeration was unusable.